### PR TITLE
Add docs for GraphQL Playground config

### PR DESCRIPTION
**Pull request by jesstelford**

### CODE REVIEW

**Comment by JedWatson** [comment]
nice, thanks @jesstelford!

**Review by JedWatson — APPROVED** [review]

File: docs/pages/docs/apis/config.mdx [hunk 1]
```diff
@@ -280,13 +280,18 @@ Options:
 - `queryLimits` (default: `undefined`): Allows you to limit the total number of results returned from a query to your GraphQL API.
   See also the per-list `graphql.queryLimits` option in the [Schema API](./schema).
 - `apolloConfig` (default: `undefined`): Allows you to pass extra options into the `ApolloServer` constructor.
-  See the [Apollo docs](https://www.apollographql.com/docs/apollo-server/api/apollo-server/#constructor) for supported options.
+  - `playground` (default: `process.env.NODE_ENV === 'development'`): If truthy, will enable the GraphQL Playground to testing queries and mutations in the browser.
```
Comment (JedWatson) [inline docs/pages/docs/apis/config.mdx:283]:
```suggestion
  - `playground` (default: `process.env.NODE_ENV === 'development'`): If truthy, will enable the GraphQL Playground for testing queries and mutations in the browser.
```

could either go with `for testing` or `to test`

**Review by JedWatson — CHANGES_REQUESTED** [review]

File: docs/pages/docs/apis/config.mdx [hunk 2]
```diff
@@ -280,13 +280,18 @@ Options:
 - `queryLimits` (default: `undefined`): Allows you to limit the total number of results returned from a query to your GraphQL API.
   See also the per-list `graphql.queryLimits` option in the [Schema API](./schema).
 - `apolloConfig` (default: `undefined`): Allows you to pass extra options into the `ApolloServer` constructor.
-  See the [Apollo docs](https://www.apollographql.com/docs/apollo-server/api/apollo-server/#constructor) for supported options.
+  - `playground` (default: `process.env.NODE_ENV === 'development'`): If truthy, will enable the GraphQL Playground for testing queries and mutations in the browser.
```
Comment (timleslie) [inline docs/pages/docs/apis/config.mdx:283]:
```suggestion
  - `playground` (default: `process.env.NODE_ENV !== 'production'`): If truthy, will enable the GraphQL Playground for testing queries and mutations in the browser.
```

File: docs/pages/docs/apis/config.mdx [hunk 3]
```diff
@@ -280,13 +280,18 @@ Options:
...
+    To configure behaviour, pass an object of [GraphQL Playground settings](https://github.com/graphql/graphql-playground#settings).
+  - See the [Apollo docs](https://www.apollographql.com/docs/apollo-server/api/apollo-server/#constructor) for more supported options.
 
 ```typescript
 export default config({
   graphql: {
     queryLimits: { maxTotalResults: 100 },
-    apolloConfig: { /* ... */ },
+    apolloConfig: {
+      playground: process.env.NODE_ENV === 'development',
```
Comment (timleslie) [inline docs/pages/docs/apis/config.mdx:292]:
```suggestion
      playground:process.env.NODE_ENV !== 'production',
```

**Review by timleslie — APPROVED** [review]
👍

**Pull request merged**